### DISCUSSION TITLE
How to connect an additional temperature sensor to BW-SHP2 v2.4?

### DISCUSSION THREAD

**Discussion by skyvladik** [arendst/Tasmota#17068] [Q&A] [answered]
The device was flashed on tasmota. Relays, LEDs, energy meter - everything works. I tried to connect the i2c sensor to pins 0 and 2 - bme280 is not displayed. I tried to connect the ds18b20 to pins 0, 2, 13 (the LED resistor was soldered off) - I did not see this sensor either, although the energy meter is displayed on the main page. I reassigned the outputs in the module configuration menu, but I did not achieve the result. Please tell me if it is possible to configure tasmota? How to do it? I don't know tasmota well.

**Comment by Jason2866** [comment, marked as answer]
**NEVER** connect anything to a power monitoring device. It is dangerous.
All wires can have AC level. For more infos please visit the Tasmota documentation.
